### DISCUSSION TITLE
Hide/show side bar with keyboard shortcut

### DISCUSSION THREAD

**Discussion by ematipico** [withastro/starlight#2264] [Feature Requests]
### What version of `starlight` are you using?

latest

### What is your idea?

Proposed here https://github.com/biomejs/website/issues/676

Not necessary

### Do you have examples of this feature in other projects?

https://rustwasm.github.io/docs/book/game-of-life/hello-world.html#whats-inside

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by Its-Just-Nans** [comment]
Upvoting

https://github.com/withastro/starlight/discussions/3559
